### PR TITLE
added new reactivate integration command

### PR DESCRIPTION
**Pull request by davidbloss**
Resolves [#506](https://github.com/OpsLevel/team-platform/issues/506)

### Checklist

- [ ] I have run this code, and it appears to resolve the stated issue.
- [ ] This PR has no user interface changes or has already received approval from product management to change the interface.
- [ ] Make a [changie](https://github.com/OpsLevel/cli/blob/main/CONTRIBUTING.md#changie-change-log-generation) entry that explains the customer facing outcome of this change

### CODE REVIEW

File: src/cmd/integration.go [hunk 1]
```diff
@@ -229,9 +229,26 @@ EOF
 	},
 }
 
+var reactivateIntegrationCmd = &cobra.Command{
+	Use:        "reactivate ID",
```
Comment (rocktavious) [inline src/cmd/integration.go:233]:
```suggestion
	Use:        "reactivate ID|ALIAS",
```

File: src/cmd/integration.go [hunk 2]
```diff
@@ -229,9 +229,26 @@ EOF
...
+	Short:      "Reactivate an integration",
+	Long:       `Reactivate an integration that was invalidated or deactivated`,
+	Example:    `opslevel update integration reactivate Z2lkOi8vb123456789`,
+	Args:       cobra.ExactArgs(1),
+	ArgAliases: []string{"ID"},
+	Run: func(cmd *cobra.Command, args []string) {
+		integration, err := getClientGQL().IntegrationReactivate(args[0])
+		cobra.CheckErr(err)
+
+		fmt.Printf("reactivated integration with id '%s'\n", integration.Id)
```
Comment (rocktavious) [inline src/cmd/integration.go:243]: Do we use this in other places or should we use the `log.Info...` way of outputting to the end user?

File: src/cmd/integration.go (hunk 1 above)
Comment (davidbloss) [reply src/cmd/integration.go:233]: I actually started with this then found out that integrations don't have aliases

File: src/cmd/integration.go (hunk 2 above)
Comment (davidbloss) [reply src/cmd/integration.go:243]: 🤔 I see a lot more instances of `fmt.Printf` than `log.Info`.

Totally, I'll add the integration name 🚀

File: src/cmd/integration.go (hunk 1 above)
Comment (rocktavious) [reply src/cmd/integration.go:233]: Ahh 😢 

But they still support `IdentifierInput` on the way in?

File: src/cmd/integration.go (hunk 2 above)
Comment (rocktavious) [reply src/cmd/integration.go]: 👍

**Review by rocktavious — APPROVED** [review]

**Pull request merged**